Sources/TuistSupport/Utils/GitHandler.swift

```diff
@@ -63,11 +71,23 @@ public final class GitHandler: GitHandling {
         }
     }
 
+    public func lsremote(url: String, tagsOnly: Bool = false, sort: String = "") throws -> String {
+        try capture(command: "git", "ls-remote", tagsOnly ? "-t" : "", !sort.isEmpty ? "--sort=\(sort)" : "", url)
+    }
```

Comment:
Do we actually need this to be so configurable? I would probably only keep the url and hardcode the others until we really need them. For example the `VersionProvider.swift` should not need to know about host to sort the stuff with the git command (i.e. the "v:refname")